docs/quickstart/species.rst

```diff
@@ -1,9 +1,36 @@
 Species properties
 ==================
+The next step is to define the chemical species present in the system. To do this, click on the `species` tab.
+On the left, you see a list of compartments in the model geometry. Select the one for which you want to define species. This species will only exist in this compartment. As a consequence, even if the species in all compartments are chemically the same, you still have to define them separately. Species only exist in the bulk volume of the compartments, i.e., species cannot be present only on the membranes.
 
-In the Species tab, the species in each compartment are listed. Clicking on a species from this list displays the species concentration and other settings.
+To create a new species, click the `Add` button, select a suitable name, and hit enter. The new species will appear below the compartment name.
 
-The initial concentration of a species can be a spatially uniform concentration, an analytic mathematical expression that may depend on the `x` and `y` location, or an image.
+On the right hand side, you see the Species Settings menu. Here you can define all relevant properties of the species.
+
+At the top, you can rename the species or assign it to a different compartment. You can also select whether it should be constant for the whole simulation or be variable. Constancy may be a useful simplifying assumption in some models if, for example, the species is produces far more quickly than the characteristic simulation timescale.
+
+.. figure:: img/species-definition.apng
+   :alt: video showing species settings
+
+   How to define a new species in a compartment together with its initial conditions and diffusion constant
+
+Two more properties of a species are of primary importance.
+
+:Initial concentration:
+   This can be set to a constant value, an analytic expression over the spatial coordinates `x` and `y` (and `z` for 3D images) or it can be derived from an image that's loaded from disk, in the .png format for 2D domains, and as .tiff file for 3D domains.
```

Comment:
```suggestion
   This can be set to a constant value, an analytic expression over the spatial coordinates `x` and `y` (and `z` for 3D images) or it can be derived from an image that's loaded from disk (any image format for 2D domains, and a stacked .tiff file for 3D domains).
```